app/views/shut_mucr_confirmation.scala.html

```diff
@@ -0,0 +1,40 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+@import controllers.routes
+@import controllers.util.FlashKeys
+
+@this(main_template: views.html.main_template)
+
+@()(implicit appConfig: AppConfig, request: Request[_], flash: Flash, messages: Messages)
+
+@main_template(
+  title = messages("shutMucrConfirmation.title"),
+  appConfig = appConfig
+) {
+
+  <div class="govuk-box-highlight">
```

Comment:
Maybe we need a component for this summary given we're adding it in for a second time, what are your thoughts?

Reply:
This is good idea. I'll add another commit with it.